### DISCUSSION TITLE
Dependabot alerts

### DISCUSSION THREAD

**Discussion by c0ze** [community/community#23681] [New to GitHub]
<p>I have some archive branches in my repo which are outdated. I dont want to delete these, but I also dont want to receive dependebot alerts for these.</p>
<p>How can I configure dependabot to run only for master ? (or main branch it is called these days ?)</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi :wave:</p>
<p>Welcome!<br>
Dependabot alerts only works on the default branch, it relies on information from the dependency graph which only scans manifests on the default branch.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="15" height="15">
      <a href="https://docs.github.com/pt/free-pro-team@latest/github/visualizing-repository-data-with-graphs/about-the-dependency-graph#about-the-dependency-graph" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="20" height="20">

<h3><a href="https://docs.github.com/pt/free-pro-team@latest/github/visualizing-repository-data-with-graphs/about-the-dependency-graph#about-the-dependency-graph" target="_blank" rel="noopener">About the dependency graph - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by c0ze** [comment]
<p>thanks, I think I got off to a wrong alarm from outdated alerts.</p>
